compare/compare.py

```diff
@@ -266,12 +266,14 @@ def get_content(distro_view="eln"):
     distro_url = "https://tiny.distro.builders"
     arches = ["aarch64", "ppc64le", "s390x", "x86_64"]
     which_source = ["source", "buildroot-source"]
-    
+
     # excludes should be changed to a URL / git repo somewhere
     # Currently they are variables in eln-periodic.py
     # For now just use a flat file
-    exclude_packagelist = open("lists/exclude.txt").read().splitlines()
-    
+    scriptpath = os.path.dirname(os.path.realpath(__file__))
+    exclude_f = os.path.join(scriptpath, "lists/exclude.txt")
```

Comment:
You should also split `lists` and `exclude.txt` into two arguments that you pass to `os.path.join`.

You could actually do that everywhere now that you're fixing these :)

Reply:
@contyk hi, updated, please review.

Reply:
:+1: